framework-plugins/lisk-framework-dashboard-plugin/src/ui/components/Button/Button.module.scss

```diff
@@ -0,0 +1,35 @@
+@import '../../styles/constants';
+
+.button {
+	display: flex;
+	flex-direction: column;
+	justify-content: center;
+	align-items: center;
+	padding: 14px 30px;
+	color: $color-white;
+	width: 167px;
+	height: 46px;
+	left: 20px;
+	top: 20px;
+	/* Ultramarine blue */
+	background: #4070F4;
+	border-radius: 3px;
```

Comment:
there are several properties that needs to be handled
https://github.com/LiskHQ/lisk-sdk/blob/development/framework-plugins/lisk-framework-faucet-plugin/src/ui/app.module.scss#L146-L169